cpp/ps0006/problem 7.cpp

```diff
@@ -80,4 +85,5 @@ void counter(int end, int start, int step) {
         std::cout << i << "\n ";
     }
     std::cout << std::endl;
-}
\ No newline at end of file
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [ ] no runtime errors, bugs or warnings while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [x] used `for` loop in a right way to to count up and down

you can check the [problem solution](https://onlinegdb.com/vdj-VXYv7) for more